packages/backend/src/services/UnfurlService/UnfurlService.ts

```diff
@@ -130,6 +278,10 @@ export class UnfurlService {
 
             await page.setExtraHTTPHeaders({ cookie });
 
+            if (chartType === ChartType.BIG_NUMBER) {
+                await page.setViewport(bigNumberViewport);
+            }
+
```

Comment:
if you don't do `else await page.setViewport(viewport);` you are going to be overwritting this bignumberviewport with the default